src/josepy/jwk.py

```diff
@@ -85,7 +85,7 @@ def _load_cryptography_key(cls, data: bytes, password: Optional[bytes] = None,
         for loader_public in (serialization.load_pem_public_key,
                               serialization.load_der_public_key):
             try:
-                return loader_public(data, backend)
+                return loader_public(data, backend)  # type: ignore[operator]
```

Comment:
Is it possible/desired to declare a type for `loader_public` to avoid this `ignore`?

Reply:
Well, with this design it is difficult to do better. I would like to add a type hint to say it is a `Callable`, but mypy still complains it is a `function`. I changed to a slightly better version that it is to say `loader_private: Any`, which is still ignoring the error, but with a proper annotation instead of disabling the error.